src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -2541,4 +2521,135 @@ public Response getFeatureFlag(@PathParam("flag") String flagIn) {
         }
     }
 
+	@GET
+	@AuthRequired
+	@Path("/datafiles/auditFiles")
+	public Response getAuditFiles(@Context ContainerRequestContext crc,
+								  @QueryParam("firstId") Long firstId, @QueryParam("lastId") Long lastId,
+								  @QueryParam("DatasetIdentifierList") String DatasetIdentifierList) throws WrappedResponse {
```

Comment:
```suggestion
								  @QueryParam("datasetIdentifierList") String datasetIdentifierList) throws WrappedResponse {
```

The above is more in keeping with the rest of the code base.